### PR TITLE
dev-util/lcov: bump to EAPI 7; add -9999

### PR DESCRIPTION
**Pull request by AnAverageHuman**

### CODE REVIEW

File: dev-util/lcov/lcov-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	SRC_URI="mirror://sourceforge/ltp/${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-linux ~x64-macos"
+fi
+
+inherit prefix
+
+DESCRIPTION="A graphical front-end for GCC's coverage testing tool gcov"
+HOMEPAGE="http://ltp.sourceforge.net/coverage/lcov.php"
+
+LICENSE="GPL-2"
```
Comment (mgorny) [inline dev-util/lcov/lcov-9999.ebuild]: Could you verify whether this isn't `GPL-2+` actually ('version 2 or (at your option) newer')?

Comment (AnAverageHuman) [reply dev-util/lcov/lcov-9999.ebuild]: I'm fairly certain the license is GPL-2. See the COPYING file in the repository: https://github.com/linux-test-project/lcov/blob/master/COPYING

Comment (mgorny) [reply dev-util/lcov/lcov-9999.ebuild]: COPYING file does not define the license for the project (even though GitHub stupidly assumes so). It is only a copy of the license, while the actual license terms are listed as copyright notices on top of source files.

Comment (AnAverageHuman) [reply dev-util/lcov/lcov-9999.ebuild]: It does seem that the license is `GPL-2+`. Will update.

File: dev-util/lcov/lcov-1.13-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+DESCRIPTION="A graphical front-end for GCC's coverage testing tool gcov"
+HOMEPAGE="http://ltp.sourceforge.net/coverage/lcov.php"
+
+LICENSE="GPL-2+"
+SLOT="0"
+
+RDEPEND="
+	dev-lang/perl
+	dev-perl/GD[png]"
```
Comment (makepost) [inline dev-util/lcov/lcov-1.13-r2.ebuild]: Could you try making `GD[png]` an optional dependency that won't be pulled if a `png` USE flag is disabled for `lcov`? Besides `genpng` and the `-F` option to `genhtml`, it doesn't seem required but the current build always wants it.

Comment (AnAverageHuman) [reply dev-util/lcov/lcov-1.13-r2.ebuild]: I've moved it into a `png` USE flag. I haven't noticed a difference, even in `genhtml` created files, so it'll be disabled by default.

Comment (mgorny) [reply dev-util/lcov/lcov-1.13-r2.ebuild]: -9999 doesn't have this.

Comment (AnAverageHuman) [reply dev-util/lcov/lcov-1.13-r2.ebuild]: Must have missed it the first time. Fixed now.

**Pull request closed without merging**